.github/workflows/build-ros2-ws-action.yml

```diff
@@ -0,0 +1,46 @@
+name: CI
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches: [master]
+jobs:
+  build-and-test:
+    name: Build and Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4.1.1
+        with:
+          path: akushon
+
+      - name: Checkout Dependencies
+        run: |
+          git clone https://github.com/ichiro-its/akushon_interfaces.git
+          git clone https://github.com/ichiro-its/kansei_interfaces.git
+          git clone https://github.com/ichiro-its/keisan.git
+          git clone https://github.com/ichiro-its/tachimawari.git
+          git clone https://github.com/ichiro-its/tachimawari_interfaces.git
+
+      - name: Setup abseil for grpc              
+        run: |
+          git clone https://github.com/abseil/abseil-cpp.git
+          cmake -S abseil-cpp -B abseil-cpp/build 
+          cmake --build abseil-cpp/build
+          sudo cmake --install abseil-cpp/build
+      
+      - name: Setup grpc        
+        run: |
+          git clone --recurse-submodules -b v1.61.0 --depth 1 --shallow-submodules https://github.com/grpc/grpc
+          cmake -S grpc -B grpc/build
+          cmake --build grpc/build
+          sudo cmake --install grpc/build
```

Comment:
Try removing this after install, hopefully it can fix this:

![imagen](https://github.com/user-attachments/assets/959a06a6-f1b2-4da5-a88b-0006c0969cd6)